relay/messages/execution.go

```diff
@@ -42,7 +42,7 @@ type ExecutionResponse struct {
 	Bundle        string      `json:"bundle"`
 	Status        string      `json:"status"`
 	StatusMessage string      `json:"status_message"`
-	Template      string      `json:"template"`
+	Template      interface{} `json:"template"`
```

Comment:
I guess we could have also used `*string` but `interface{}` was a bit nicer to work with.